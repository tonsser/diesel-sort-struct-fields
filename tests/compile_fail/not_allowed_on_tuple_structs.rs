use diesel_sort_struct_fields::sort_fields;

#[sort_fields]
pub struct Thing(i32, i32);

fn main() {}
