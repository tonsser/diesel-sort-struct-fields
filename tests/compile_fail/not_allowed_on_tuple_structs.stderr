error: `#[sort_fields]` is not allowed on tuple structs, only structs with named fields
 --> $DIR/not_allowed_on_tuple_structs.rs:4:17
  |
4 | pub struct Thing(i32, i32);
  |                 ^^^^^^^^^^
